docs/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM squidfunk/mkdocs-material
```

Comment:
This won't appear in the generated docs somehow by mistake?

Reply:
It doesn't show up but it can be downloaded.

I can exclude it.